dotcom-rendering/src/lib/liveblogAdSlots.ts

```diff
@@ -68,94 +75,166 @@ type KnownBlockElementType =
 
 /**
  * Approximations of the height of each type of block element in pixels.
- * Predictions are made for MOBILE viewports, as data suggests that the
- * majority of liveblog page views are made using mobile devices.
+ * "...Mobile" indicates the value is used for devices up to the tablet breakpoint."
  */
 const elementHeightDataMap: {
 	[key in KnownBlockElementType]: BlockElementHeightData;
 } = {
 	'model.dotcomrendering.pageElements.BlockquoteBlockElement': {
 		heightExcludingText: 0,
+		heightExcludingTextMobile: 0,
+		margin: 16,
 		textHeight: {
-			lineHeight: 25.5,
+			lineHeight: 23.8,
```

Comment:
These height changes are the result of running a tool in the commercial tools repo that looks to find what inputs can accurately predict the height of an element.

This tool is not yet in the `main` branch of the commercial tools repo, but will be added shortly.